react/features/analytics/functions.js

```diff
@@ -76,6 +78,9 @@ export function initAnalytics({ getState }: { getState: Function }) {
             }
 
             analytics.addPermanentProperties(permanentProperties);
+            analytics.setConferenceName(permanentProperties.roomName);
```

Comment:
I really like that this is a separate explicit call. Maybe it was always like that but I like the approach that addPermanentProperties doesn't create additional side effects.